### ISSUE TITLE
Geohash: Sort on other things than tf-idf?

### ISSUE THREAD

**Issue by eklem** [fergiemcdowall/search-index#476]
Is it possible with v1.0.x to sort on other things than tf-idf. I was thinking of making a Geohash example: http://geohash.org/site/tips.html

**Comment by fergiemcdowall** [comment]
Yes sorting is much better in v1.0.x., and a geohash example would be really cool.

Will sort out the docs on this when I have time

**Comment by eklem** [comment]
I guess what's left here is that I test the [geo-search-helper](https://github.com/eklem/geo-search-helper) and document a little how a numerical sorting on distance can be used. In the end I didn't use the geohash library, but think it will work well without. It had some issues that made it a bit clunky.
